playwright/pages/dialogs/addNavigationMenuItemDialog.ts

```diff
@@ -14,20 +14,37 @@ export class AddNavigationMenuItemDialog extends BasePage {
     super(page);
   }
 
-  async selectLinkOption(option: string, optionName: string) {
+  async selectLinkTypeOption(linkType: string) {
     await this.menuLinkType.click();
     await this.waitForDOMToFullyLoad();
-    await this.menuLinkOptions.filter({ hasText: "Categories" }).waitFor({ state: "visible" });
-    await this.menuLinkOptions.filter({ hasText: "Collections" }).waitFor({ state: "visible" });
-    await this.menuLinkOptions.filter({ hasText: "Pages" }).waitFor({ state: "visible" });
-    await expect(this.menuLinkOptions.filter({ hasText: option })).toBeEnabled();
-    await this.menuLinkOptions.filter({ hasText: option }).click({ force: true });
-    await this.waitForDOMToFullyLoad();
+
+    // Ensure the link type option is visible and select it
+    const linkTypeOption = this.menuLinkOptions.filter({ hasText: linkType });
+
+    await linkTypeOption.waitFor({ state: "visible" });
+    await expect(linkTypeOption).toBeEnabled();
+    await linkTypeOption.click({ force: true });
+
+    // Verify the correct link type is selected
+    const selectedLinkType = await this.menuLinkType.inputValue();
+
+    if (selectedLinkType !== linkType) {
+      throw new Error(`Expected link type "${linkType}" but found "${selectedLinkType}"`);
```

Comment:
this should be an assert, not exception raising, this will break the test execution by unhandled error